src/meson.build

```diff
@@ -61,9 +61,12 @@ endif
 
 
 cxx_flags_asan = run_command('/bin/sh', '-c', 'echo $CXXFLAGS $CPPFLAGS | grep fsanitize')
-if get_option('b_sanitize').contains('address') or cxx_flags_asan.returncode() == 0
-  message('Address sanitizer enabled, disabling internal backtrace')
-  debug_arguments += ['-DASAN_ENABLED']
+if (not (get_option('b_sanitize').contains('address') or cxx_flags_asan.returncode() == 0)
```

Comment:
I find this a bit confusing, took me awhile to understand what the ifs do exactly. Can't we just set `print_trace` to false if asan is enabled, and then add the debug arguments in a second if?

Reply:
Sorry for the delay